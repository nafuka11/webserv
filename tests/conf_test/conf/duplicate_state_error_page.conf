server {
    location /error_page {
        error_page 404 /404.html;
        error_page 501 502 503 /50x.html;
        error_page 404 /404.html;
    }
}