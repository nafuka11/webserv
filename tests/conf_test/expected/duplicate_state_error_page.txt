webserv: duplicate "error_page:404" value. This value is already set. in ./conf/duplicate_state_error_page.conf:5
